Platform/obc/hal/uhf_hal.c

```diff
@@ -0,0 +1,374 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file uhf_hal.c
+ * @author Arash Yazdani
+ * @date 2020-10-26
+ */
+
+/*
+ * When TRX connected, the stubbed blocks can be used for TRX = off situation.
+ */
+
+#include "uhf_hal.h"
+#include "hal.h"
+#include "services.h"
+#include "queue.h"
+#include <stdio.h>
+#include <string.h>
+
+static UHF_Status U_status_reg = {.uptime = 12, .pckts_out = 100, .pckts_in = 70, .pckts_in_crc16 = 10,
+                                  .temperature = 18.4, .firmware_ver = 2, .payload_size = 127, .secure_key = 32};
```

Comment:
The initialization for the test purposes since these parameters aren't set and are more HK stuff.

Reply:
Ok, maybe wrap it in the compiler directive, and put a comment in to make it clear that these values are arbitrary, and for testing

Reply:
Sounds good.